compiler/x/codegen/CompareAnalyser.cpp

```diff
@@ -730,8 +730,7 @@ void TR_X86CompareAnalyser::longOrderedCompareAndBranchAnalyser(TR::Node       *
    cg()->decReferenceCount(firstChild);
    cg()->decReferenceCount(secondChild);
 
-   if (deps)
-      deps->setMayNeedToPopFPRegisters(true);
+   deps->setMayNeedToPopFPRegisters(true);
```

Comment:
This isn't the only occurrance of this in the method that I can see so we should either fix them all or leave it along IMO.